### DISCUSSION TITLE
What is the current position on lerna pre-release?

### DISCUSSION THREAD

**Discussion by mkrei** [yarnpkg/berry#3980] [Q&A] [answered]
Hi all,
some time ago, [a bug](https://github.com/yarnpkg/berry/issues/1445) was reported regarding failed installation of pre-releases created via lerna when running yarn2. 

I'm now running into the same issue in a repository using yarn3, when attempting to integrate a pre-release created via lerna:

`private-package@npm:1.2.3-alpha.5+coffeef: Registry failed to return reference "1.2.3-alpha.5"`

Has there ever been a conclusion to this issue? As I understand from the other reports on the closed bug report, the issue itself remains unresolved, but there is also no clear indication on which steps to take or a statement that it's just not a use-case supported by yarn berry

**Comment by merceyz** [comment, marked as answer]
Hey, thanks for the reminder about that bug!
I opened https://github.com/yarnpkg/berry/pull/4008 to fix the issue, you can try it out using
```sh
yarn set version from sources --branch 4008
```

**Reply by mkrei** [reply]
Thanks for looking into this, with the changes from #4008 everything works just fine.

Can you by chance already make any guesses on when this can be expected as part of an official release?

**Reply by arcanis** [reply]
Soon, just a couple of things I want to check first. In any case, beta releases are usually just as good as stable.

**Reply by merceyz** [reply]
@mkrei I created a canary release so you can find the fix in `3.2.0-rc.12`
```sh
yarn set version canary
```
